smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/ServiceClientConfigGenerator.kt

```diff
@@ -77,8 +77,9 @@ class ServiceClientConfigGenerator(
      * Derives client config properties from the service context params trait.
      */
     private fun clientContextConfigProps(trait: ClientContextParamsTrait): List<ConfigProperty> = buildList {
-        trait.parameters.forEach { (k, v) ->
-            add(
+        trait
+            .parameters
+            .map { (k, v) ->
```

Comment:
**Correctness**: Remove `buildList`. As written, this always returns an empty list.

Reply:
**_NICE_** catch!